packages/gatsby/src/redux/actions/public.ts

```diff
@@ -1112,21 +1143,25 @@ actions.replaceWebpackConfig = (config: Object, plugin?: ?Plugin = null) => {
  *   }
  * })
  */
-actions.setBabelOptions = (options: Object, plugin?: ?Plugin = null) => {
+actions.setBabelOptions = (
+  options: Record<string, any>,
```

Comment:
I would just make this `object`. Record is only useful if you are specifying constraints

Reply:
If it is `object` the following line makes TS angry:

```javascript
if (typeof options.options !== `object`) {
```

`Property 'options' does not exist on type 'object'.`

Reply:
Yeah, I take back my comment anyhow. Record is actually a good util :)